web/cobrands/zurich/layout.scss

```diff
@@ -10,7 +10,6 @@ $mappage-header-height: 7em;
 // White background, so no shadow or margin needed.
 .content {
     color: #3c3c3c;
-    @include box-shadow(none);
 }
 .iel8 {
     .content {
```

Comment:
And just below, body.mappage .content having anything (a box-shadow here) is meaningless now, as it only contains absolutely positioned map + sidebar?